attributes/events/dispatches.md

```diff
@@ -0,0 +1,32 @@
+# Dispatches
+
+The `Dispatches` attribute allows you to dispatch custom (Laravel Events)[https://laravel.com/docs/10.x/events#defining-events].
+
+
+## Usage
+
+```php
+use Illuminate\Database\Eloquent\Model;
+use WendellAdriel\Lift\Attributes\Events\Observer;
+use WendellAdriel\Lift\Lift;
+
+
+#[Dispatches(ProductSaved::class)]
+#[Dispatches(ProductHasBeenSetup::class, 'created')]
+final class Product extends Model
+{
+    use Lift;
+
+}
+```
+
+## API
+
+```php
+#[Dispatches(string $eventClass, string $event = '')]
+```
+
+Properties:
```

Comment:
Put this and below in text format not list, because you don't need to add the name or the type of the parameters since it's already described above in the API example.